core/src/main/java/hudson/slaves/RetentionStrategy.java

```diff
@@ -33,8 +33,10 @@
 import java.util.Collections;
 import java.util.HashMap;
 import jenkins.model.Jenkins;
+import net.jcip.annotations.GuardedBy;
```

Comment:
this import does not match what is is use throught the rest of Jenkins core (be it correct or incorrect).